server/schemas/internalapi/v1/schema.proto

```diff
@@ -24,6 +24,7 @@ service ReEarthCMS {
   rpc GetModelGeoJSONExportURL(ExportRequest) returns (ExportURLResponse) {
     option deprecated = true;
   }
+  rpc PatchStarCount(PatchStarCountRequest) returns (PatchStarCountResponse) {}
```

Comment:
```suggestion
  rpc StarProject(StarRequest) returns (StarResponse) {}
  rpc UnstarProject(StarRequest) returns (StarResponse) {}
```
I assume that the user can star the project and remove his own star when he need to

Reply:
done. We might want to use one method to 'star' and 'unstar' as the logic to know which endpoint to use might need to backend.